apps/developer-hub/src/components/HomepageHero/HeroContent/index.module.scss

```diff
@@ -0,0 +1,59 @@
+@use "@pythnetwork/component-library/theme";
+
+.heroContent {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  @include theme.breakpoint(md) {
+    grid-column: 1 / 2;
+  }
+
+  @include theme.breakpoint(lg) {
+    grid-column: 1 / 3;
+  }
+}
+
+.title {
+  font-size: theme.spacing(6);
+  line-height: 1.2;
+  font-weight: theme.font-weight(bold);
+  margin-bottom: theme.spacing(3);
+  color: theme.color("foreground");
+
+  @include theme.breakpoint(md) {
+    font-size: theme.spacing(8);
+  }
+}
+
+.subtitle {
+  font-size: theme.spacing(4);
+  line-height: 1.5;
+  margin-bottom: theme.spacing(6);
+  color: theme.color("muted");
+
+  @include theme.breakpoint(md) {
+    font-size: theme.spacing(5);
+  }
+}
+
+.ctaGroup {
+  display: flex;
+  flex-direction: column;
+  gap: theme.spacing(4);
+
+  @include theme.breakpoint(sm) {
+    flex-direction: row;
+    align-items: center;
+  }
+}
+
+.secondaryLink {
+  color: theme.color("foreground");
+  text-decoration: none;
```

Comment:
In general, [wcag requirements state that links must be discernable](https://dequeuniversity.com/rules/axe/4.10/link-name) in a very obvious way using features other than color.  I see you adjusted the font weight which may be sufficient, but underlining text is the most effective and accessible mechanism for satisfying this requirement, so I generally recommend against suppressing the underline.

If you do choose to move forward with this, you can pass [the invert prop](https://github.com/pyth-network/pyth-crosschain/blob/main/packages/component-library/src/Link/index.tsx#L9) to the Link component so you don't have to override the style manually.